package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -0,0 +1,777 @@
+            # Generate frame_results
+            frame_results = []
+            for s1_frame_results, s2_frame_results in self._water_network.values():
+                frame_results.extend(list(s1_frame_results))
+                frame_results.extend(list(s2_frame_results))
+
+            self._timeseries.append(frame_results)
+
+        logger.info("WBridge analysis: complete; timeseries  %s.timeseries",
+                    self.__class__.__name__)
+    @staticmethod
+    def _reformat_hb(hb, atomformat="{0[0]!s}{0[1]!s}:{0[2]!s}"):
+        """
+        .. deprecated:: 1.0
+            This is a compatibility layer so the water bridge
+            _timeseries to timeserie conversion can perform as it does in the hydrogen
+            bond analysis.
+
+            For more information about the function of _reformat_hb in hydrogen
+            bond analysis, please refer to the _reformat_hb in the hydrogen
+            bond analysis module.
+
+            As the _timeseries to timeserie conversion will be deprecated in 1.0
+            this function will automatically lose its value.
+        """
+
+        return (list(hb[:2])
+                + [atomformat.format(hb[2]), atomformat.format(hb[3])]
+                + list(hb[4:]))
+
+    @property
+    def timeseries(self):
+        r'''Time series of water bridges.
+
+        Due to the intrinsic complexity of water bridge problem, where several
+        atoms :math:`n` can be linked by the same water. A simple ``atom 1 - water bridge - atom 2`` mapping will create a very large list
+        :math:`n!/(2(n-2)!)` due to the rule of combination.
+
+        Thus, the output is arranged based on each individual bridging water
+        allowing the later reconstruction of the water network. The hydrogen
+        bonds from selection 1 to the **first** bridging water is followed by
+        the the hydrogen bonds from selection 2 to the **same** bridging water.
+        After that, hydrogen bonds from selection 1 to the **second** bridging
+        water is succeeded by hydrogen bonds from selection 2 to the **same
+        second** bridging water. An example of the output is given in
+        :ref:`Analysis Output` .
+
+        Note
+        ----
+        To find an acceptor atom in :attr:`Universe.atoms` by
+        *index* one would use ``u.atoms[acceptor_index]``.
+
+        The :attr:`timeseries` is a managed attribute and it is generated
+        from the underlying data in :attr:`_timeseries` every time the
+        attribute is accessed. It is therefore costly to call and if
+        :attr:`timeseries` is needed repeatedly it is recommended that you
+        assign to a variable::
+
+           w = WaterBridgeAnalysis(u)
+           w.run()
+           timeseries = w.timeseries
+
+        See Also
+        --------
+        :attr:`table` : structured array of the data
+        '''
+        HydrogenBondAnalysis.timeseries(self)
```

Comment:
This is not returning anything. How is this supposed to work??

Reply:
Also, see next comment with`super()`.

Reply:
@orbeckst Sorry, in my original tested version, I didn't write a separate timeseries method. I write this method when writing the docs but forget to test it.